Cython/Coverage.py

```diff
@@ -50,13 +50,23 @@ def _find_dep_file_path(main_file, file_path, relative_path_search=False):
     return canonical_filename(abs_path)
 
 
+class NoCoverConfigurer(CoveragePlugin):
+    exclude_lines = []
+
+    def configure(self, config):
```

Comment:
Can't the `Plugin` implement this method itself, in order to receive the list of exclusion regexes? Does it need to be a separate plugin?

Reply:
The "lines" are indeed regexes. I am not sure why they call it that but the naming is that of `coverage` so that we should just use it. It is most likely possible to implement the `configure()` method on `Plugin` and to register it both as a `configurer` and a `file_tracer`. However, I would rather keep the responsibilities separated.

In the end, I am only interested in the functionality. So feel free to modify this code.

Reply:
> I am not sure why they call it that

The name reads well when used in a config file, referring to "exclude lines … that look like these". It makes less sense when used in code that processes the entries, where it suggests a reference to specific lines or line numbers. I needed to look up the documentation of coverage plugins in order to understand this implementation. That suggests that something was lacking from the code, be it in terms of comments or naming.

> I would rather keep the responsibilities separated.

If it's at the price of globally storing duplicate configuration that coverage stores internally, I'd rather not.

In the end, it's mostly a style issue. I'll revert my last change and merge this, then see what else I can do.

Reply:
> In the end, it's mostly a style issue. I'll revert my last change and merge this, then see what else I can do.

If you do not mind, please ping me on this. `coverage` is well documented and the code is rather understandable. But it is lacking on examples. I would be genuinely interested in following up with this.